config.go

```diff
@@ -46,7 +46,10 @@ For example, to allocate and mount an audio capture device: {"name": "capture",
 For example, to expose a CH340 serial converter: {"name": "ch340", "groups": [{"usb": [{"vendor": "1a86", "product": "7523"}]}]}
 A "count" can be specified to allow a discovered device group to be scheduled multiple times.
 For example, to permit allocation of the FUSE device 10 times: {"name": "fuse", "groups": [{"count": 10, "paths": [{"path": "/dev/fuse"}]}]}
-Note: if omitted, "count" is assumed to be 1`)
+Note: if omitted, "count" is assumed to be 1
+If mountPath is a directory, the device will be mounted to the directory with the name of the device.
+For example, to expose the serial devices to the /dev/serial directory: {"name": "serial", "groups": [{"paths": [{"path": "/dev/ttyUSB*", "mountPath": "/dev/serial/"}]}]}
+`)
```

Comment:
This is causing the linter to fail because the README.md is expecting to have 1 more newline than you commit includes. If you remove this, then linting should pass 👍 
```suggestion
For example, to expose the serial devices to the /dev/serial directory: {"name": "serial", "groups": [{"paths": [{"path": "/dev/ttyUSB*", "mountPath": "/dev/serial/"}]}]}`)
```